src/plot_api/plot_api.js

```diff
@@ -1961,6 +1961,7 @@ function _relayout(gd, aobj) {
         else {
             // check whether we can short-circuit a full redraw
             // 3d or geo at this point just needs to redraw.
+            var pp1 = String(p.parts[1] || '');
```

Comment:
Can you move this new line above the `check whether we can short-circuit` comment?